example/src/App.js

```diff
@@ -181,6 +181,7 @@ const App = () => {
         isoCode='ES'
         onLinkOpen={onLinkOpen}
         isLoading={isLoading}
+        validateJSON
```

Comment:
We should avoid truly expressions. The reason behind this is if I'm passing to boolean expressions let say `validateJSON` and `notValidateJSON` I would not know the behavior without going to the implementation details. Think components as functions, passing booleans as arguments makes the function really hard to maintain. I'd use some sort of state handling @sofisdev